packages/client-core/src/common/services/LocationInstanceConnectionService.ts

```diff
@@ -85,51 +71,38 @@ export function useWorldInstance() {
   return worldHostId.value ? worldInstanceState[worldHostId.value] : null
 }
 
-export const LocationInstanceConnectionServiceReceptor = (action) => {
+export const changeActiveConnectionHostId = (currentInstanceId: UserId, newInstanceId: UserId) => {
```

Comment:
`changeActiveConnectionHostId` and `provisionLocationInstanceServer` need to be added to `LocationInstanceConnectionService` instead of top level exports